debian/control

```diff
@@ -107,7 +107,7 @@ Build-Depends: automake,
 # Make-Check   xmlstarlet,
                nasm [amd64],
                zlib1g-dev,
-# Jaeger       Built-Using:   libyaml-cpp-dev (>= 0.6),
+# Jaeger Built-Using:   libyaml-cpp-dev (>= 0.6),
```

Comment:
@clwluvw could you be more specific on "fix"?

Reply:
@tchaikov Here https://github.com/ceph/ceph/blob/master/install-deps.sh#L68 when removing `# Jaeger ` from this line `debian/control` file will be like this:
```
...
# Make-Check   xmlstarlet,
               nasm [amd64],
               zlib1g-dev,
      Built-Using:   libyaml-cpp-dev (>= 0.6),
...
```
and it's not parsable!

Reply:
@clwluvw i think a better way is to fix the regular expression used in `install-deps.sh`. for instance, 

```
sed -i -e 's/^# Jaeger[[:space:]]*//g' $control
```

@ideepika how did you test this change? i am wondering if you pushed an old branch?

Reply:
@tchaikov  thanks for the ping, missed the notification
@tchaikov I used centos to test these, I missed catching them.
@clwluvw we need to also remove `#crimson libyaml-cpp` if we are using yaml-cpp that supports jaeger, otherwise there would be build failures due to linking to older version of libyaml. 
`#Jaeger  ` would also need some indentation here for debian/control to parse correctly, can you update changes considering these points, something like: 
https://github.com/ceph/ceph/pull/38783/files#diff-1335d27b188d672bf9a9b572f40df7c4699b8d9998fb16ba284f4acb4e77077cR74

Reply:
@ideepika The first part has been done in #38743. Could you please first review and merge that?

Reply:
@clwluvw https://github.com/ceph/ceph/pull/38743/files#r580288000 and it would be ideal if you can amend all in one PR, easier to review :)

Reply:
> @clwluvw https://github.com/ceph/ceph/pull/38743/files#r580288000 and it would be ideal if you can amend all in one PR, easier to review :)

@tchaikov, this needs to be addressed, I couldn't get back to complete this pr: https://github.com/ceph/ceph/pull/38743/files#r580288000 will do if @clwluvw is not working on it

Reply:
@clwluvw hi Seena, are you still working on this change?

Reply:
@tchaikov @ideepika sorry for the delay. I've made an update here: https://github.com/ceph/ceph/pull/38743